src/client/components/definition/definitionData.js

```diff
@@ -3,29 +3,34 @@ module.exports = {
     return [
       {
         title: 'Dividend (Yield %)',
-        definition: 'An amount of the company\'s profits which is paid to the ' +
-        'holders of that company\'s stock. Dividend is paid as a certain amount per share. ' +
-        'Dividend yield is a percentage calculated as (dividend per share / price per share) * 100%.'
+        definition: 'An amount of the company\'s profits which is paid to ' +
+        'the holders of that company\'s stock. Dividends are paid as a ' +
+        'certain amount of money per share. Dividend yield is a percentage, ' +
+        'calculated as (dividend per share / price per share) * 100%.'
       },
       {
         title: 'Free Cash Flow Yield (abbrev. as FCFY)',
-        definition: 'A comparison between free cash flow per share and share price. ' +
-        'Higher percentages mean the company is generating more free cash flow per share, which is appealing. ' +
-        'Calculated as (free cash flow per share / market price per share) * 100%.'
+        definition: 'A comparison between free cash flow per share and ' +
+        ' share price. Higher percentages mean the company is generating ' +
+        'more free cash flow per share, which is appealing. Calculated as ' +
+        '(free cash flow per share / market price per share) * 100%.'
       },
       {
         title: 'High',
-        definition: 'Highest price at which the security was traded during the trading day.'
+        definition: 'Highest price at which the security was traded during ' +
+        'the trading day.'
       },
       {
         title: 'Low',
-        definition: 'Lowest price at which the security was traded during the trading day.'
+        definition: 'Lowest price at which the security was traded during ' +
+        'the trading day.'
       },
       {
         title: 'Market Cap',
         definition: 'Market capitalization (market cap) is the total ' +
-        'market value of a company\'s outstanding shares. ' +
-        'To calculate market cap, multiply the company\'s number of outstanding shares by it\'s share price.'
+        'market value of a company\'s outstanding shares. To calculate ' +
+        'market cap, multiply the company\'s number of outstanding shares ' +
+        'by it\'s share price.'
```

Comment:
not a change on this branch but since you're already fixing this file, it should be "its", not "it's"